app/routes/__index/quest/index.tsx

```diff
@@ -0,0 +1,58 @@
+import PlusIconWhite from "~/components/Icons/PlusIcon";
+import { LinkFooter } from "~/components/Buttons/LinkButton";
+import Carousel from "~/components/Quest/Carousel";
+import { db } from "~/utils/db.server";
+import { useLoaderData } from "@remix-run/react";
+import { json } from "@remix-run/node";
+
+
+
+
+export async function loader() {
+    const allProjects = await db.badges.findMany({
+        orderBy: [
+          {
+            token_id: 'desc',
+          }
+        ]});
+
+    return json(allProjects);
+};
+
+
+export default function Quest() {
+    const badges = useLoaderData();
```

Comment:
badges should have the Badge[] type from prisma.

Reply:
![image](https://user-images.githubusercontent.com/25151724/209676033-f724323a-78c5-4fcb-b13e-ec545f89f8bd.png)
done like this ok ?